plugins/modules/route53_crrs.py

```diff
@@ -0,0 +1,504 @@
+#!/usr/bin/python
+# This file is part of Ansible
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+from ansible_collections.amazon.aws.plugins.module_utils.aws.core import (
+    AnsibleAWSModule,
+)
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: route53_crrs
+short_description: create, update or delete route53 resource record sets.
+description: create, update or delete route53 resource record sets.
+requirements: [boto3, botocore, python>=3.5]
+options:
+  action:
+    description: >
+      - The action to perform:
+        CREATE  Creates a resource record set that has the specified values.
+        DELETE  Deletes a existing resource record set.
+        UPSERT  If a resource record set doesn't already exist, Route 53
+                creates it. If a resource record set does exist, Route 53
+                updates it with the values in the request.
+    required: true
+    choices: ['CREATE', 'DELETE', 'UPSERT']
+    type: str
+  name:
```

Comment:
Why not `zone`/`hosted_zone_id` and `record`, as for the `route53` module? Or at least have that as an alternative to `name`.